src/pages/privacy/index.tsx

```diff
@@ -99,8 +99,9 @@ export default function PrivacyPolicy() {
         </Typography>
         <StyledArticleBody variant="body1" paragraph>
           We use your search and the associated query and context to provide you with the correct ranking of the search
-          results. We also use your Google Drive App information storage to provide multi-device course schedule
-          synchronization.
+          results. If you are logged in, we will also use your user ID to store selected courses between multiple
+          devices. Users are also required to login using their student email before writing a course review to confirm
```

Comment:
```suggestion
          devices. Users are also required to login using their student email before writing and interaction with course reviews, to confirm
```
So it includes like, dislike, report

Reply:
```suggestion
          devices. Users are also required to login using their student email before writing and interacting with course reviews, to confirm
```